src/main/java/cms/gov/madie/measure/utils/MeasureUtil.java

```diff
@@ -78,15 +87,15 @@ public Measure validateAllMeasureDependencies(Measure measure) {
     return measureBuilder.build();
   }
 
-  private boolean isCqlDefsToSupplementalDataMismatched(Measure measure, String elmJson) {
+  private boolean isCqlDefsaMismatched(List<DefDescPair> defDescPairs, String elmJson) {
     boolean result = false;
-    if (CollectionUtils.isEmpty(measure.getSupplementalData())) {
+    if (CollectionUtils.isEmpty(defDescPairs)) {
       result = false;
     } else {
       result =
-          !measure.getSupplementalData().stream()
+          !defDescPairs.stream()
               .anyMatch(
-                  sde -> cqlDefinitionReturnTypeValidator.validateSdeDefinition(sde, elmJson));
+                  sde -> cqlDefinitionReturnTypeValidator.validateDefDescription(sde, elmJson));
```

Comment:
The inner variable can be more generic since the Collection handles any definition type.
```suggestion
                  def -> cqlDefinitionReturnTypeValidator.validateDefDescription(def, elmJson));
```